appengine/standard/firebase/firetactoe/firetactoe.py

```diff
@@ -94,25 +103,30 @@ def create_custom_token(uid, valid_minutes=60):
     security rules to prevent unauthorized access. In this case, the uid will
     be the channel id which is a combination of user_id and game_key
     """
-    header = base64.b64encode(json.dumps({'typ': 'JWT', 'alg': 'RS256'}))
```

Comment:
I'd prefer to leave this where it is. Using the app identity service is a google-specific feature that I wanted the user to grok before getting lost in the guts of generating a JWT.